vllm/v1/sample/rejection_sampler.py

```diff
@@ -9,18 +10,45 @@
 from vllm.platforms import current_platform
 from vllm.v1.outputs import SamplerOutput
 from vllm.v1.sample.metadata import SamplingMetadata
+from vllm.v1.sample.ops.topk_topp_sampler import random_sample
```

Comment:
nit: Please move this out to `sample/ops/utils.py` or something like that. It looks a bit awkward that rejection sampler imports the top-p sampler just for this method.